R/imports-env.r

```diff
@@ -120,23 +124,3 @@ onload_assign("process_imports", {
 
   process_imports
 })
-
```

Comment:
I think you can also remove the corresponding source for `load_namespace_for1()` and friends.

Reply:
Nope, my mistake,, they're also used elsewhere.